integration/pipelines_test.go

```diff
@@ -95,6 +96,36 @@ var _ = Describe("Fly CLI", func() {
 					}))
 				})
 			})
+
+			Context("completion", func() {
+				BeforeEach(func() {
+					os.Setenv("GO_FLAGS_COMPLETION", "1")
+					atcServer.AppendHandlers(
+						ghttp.CombineHandlers(
+							ghttp.VerifyRequest("GET", "/api/v1/teams/main/pipelines"),
+							ghttp.RespondWithJSONEncoded(200, []atc.Pipeline{
+								{Name: "some-pipeline-1", URL: "/pipelines/some-pipeline-1", Paused: false, Public: false},
+								{Name: "some-pipeline-2", URL: "/pipelines/some-pipeline-2", Paused: false, Public: false},
+								{Name: "another-pipeline", URL: "/pipelines/another-pipeline", Paused: false, Public: false},
+							}),
+						),
+					)
+				})
+
+				AfterEach(func() {
+					os.Unsetenv("GO_FLAGS_COMPLETION")
+				})
+
+				It("returns all matching pipelines", func() {
+					flyCmd := exec.Command(flyPath, "-t", targetName, "get-pipeline", "-p", "some-")
```

Comment:
I'd like to prevent doing `os.Setenv` and `os.Unsetenv` if at all possible.

I'm pretty sure that [exec.Cmd](https://golang.org/pkg/os/exec/#Cmd) supports setting the environment (`.Env`) of the command before it runs, can you change this to do that instead?